src/schema/v2/sales.ts

```diff
@@ -31,30 +31,38 @@ export const SalesConnectionField: GraphQLFieldConfig<void, ResolverContext> = {
     isAuction: {
       description: "Limit by auction.",
       type: GraphQLBoolean,
-      defaultValue: true,
     },
     live: {
       description: "Limit by live status.",
       type: GraphQLBoolean,
-      defaultValue: true,
     },
     published: {
       description: "Limit by published status.",
       type: GraphQLBoolean,
-      defaultValue: true,
     },
     sort: SaleSorts,
   }),
   resolve: async (
     _root,
     { ids, isAuction, live, published, sort, ...paginationArgs },
-    { salesLoaderWithHeaders }
+    {
+      userRoles,
+      unauthenticatedLoaders: { salesLoaderWithHeaders: loaderUnauthenticated },
+      authenticatedLoaders: { salesLoaderWithHeaders: loaderAuthenticated },
+    }
   ) => {
+    paginationArgs.first = paginationArgs.first || 20
     const { page, size, offset } = convertConnectionArgsToGravityArgs(
       paginationArgs
     )
+    let loader
+    if (userRoles.includes("team") && ids.length) {
```

Comment:
This winds up being future proof for Positron users to be `team` (and not necessarily `admin`).

However, Gravity still only respects `admin` as a role to return privileged content. If there's still a need for the Positron administrative UI to display unpublished sales (for example), the users would still need to have the `admin` role unless Gravity had an update.

Reply:
For the moment I believe that is ok -- but we should do some research to see who is using this feature in Positron. At the moment all staff has admin roles. For the future, I believe it is non-editorial people from fairs/auctions/galleries that are using this UI and potentially there are other roles to target.